.github/actions/resolve-rust-versions/resolve.sh

```
@@ -0,0 +1,132 @@
+fi
+
+# Beta doesn't have Docker images, so we just pass through the version
+# It will be installed via rustup in the workflow
+RESOLVED_BETA="$RUSTUP_BETA"
+
+echo "Resolved versions:"
+echo "  stable: $RESOLVED_STABLE"
+echo "  beta: $RESOLVED_BETA"
+echo "  stable-docker-available: $STABLE_DOCKER_AVAILABLE"
+
+echo "stable=$RESOLVED_STABLE" >>"$GITHUB_OUTPUT"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)